packages/meta-harness/scripts/meta_harness.py

```
@@ -547,18 +619,31 @@ def build_parser() -> argparse.ArgumentParser:
-    for stub_name in _PHASE_1B_STUBS:
-        _add_common_args(
-            sub.add_parser(stub_name, help=f"（Phase 1b/2 未実装スタブ: {stub_name}）")
-        )
+    p_evaluate = sub.add_parser("evaluate", help="候補をシナリオ評価する")
+    _add_common_args(p_evaluate)
+    p_evaluate.add_argument("--candidate", required=True, help="評価対象の cand_id")
+    p_evaluate.add_argument(
+        "--scenario",
+        action="append",
+        default=None,
+        help="評価するシナリオ id（複数指定可、省略時は suite 内の全シナリオ）",
```

Ruff (0.15.20)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)